db/smt-db

```diff
@@ -304,7 +305,7 @@ sub setup
     my ($login,$pass,$uid,$gid) = getpwnam("wwwrun");
     chown 0, $gid, "/etc/smt.conf";
 
-    `smt-sql --select-mode-direct /usr/share/schemas/smt/mysql/2.03/100-smt-tables.sql > /var/log/smt/smt-db-setup.log`;
+    `smt-sql --select-mode-direct /usr/share/schemas/smt/mysql/3.00/100-smt-tables.sql > /var/log/smt/smt-db-setup.log`;
```

Comment:
- remove all directors with only a single number
- create a new directory named "latest" or "current" and put the 100-smt-tables there.
- for a new setup always call /usr/share/schemas/smt/mysql/latest/100-smt-tables.sql

If we change the schema in future, we create a migration directory <oldversion>-<new-version> and update 
latest/100-smt-tables.sql

This is my idea.